doc/z-chap1.xml

```diff
@@ -94,6 +94,45 @@ make]]></Listing>
     If you want to check that the package is working correctly, you
     should run some of the tests described in Section
     <Ref Sect="Testing your installation"/>.
+    <Subsection Label="Configuration options">
+      <Heading>Configuration options</Heading>
+
+In addition to the usual autoconf generated configuration flags, the following
+flags are provided.
+
+<Table Align="|l|l|">
+ <Caption>Configuration flags</Caption>
+ <HorLine/>
+  <Row>
+   <Item>Option</Item><Item>Meaning</Item>
+  </Row>
+ <HorLine/>
+ <HorLine/>
+ <Row>
+<Item>--enable-code-coverage        </Item><Item> enable code coverage support</Item>
```

Comment:
I think the flags would look better in `<C>` tags but whatevs 🙂 
```suggestion
<Item><C>--enable-code-coverage</C></Item><Item> enable code coverage support</Item>
```

Reply:
Agreed, looks swish in text mode :)